modules/ROOT/pages/key-configurations.adoc

```diff
@@ -0,0 +1,11 @@
+= Configuring Behavior during Runtime
```

Comment:
```suggestion
= Configuring Behavior During Runtime
```

Reply:
According to the https://capitalizemytitle.com/style/Chicago/ site, "during" should not be capitalized. 
Please, can you confirm?
Thanks!

Reply:
During should be capitalized because it's a preposition of more than 5 letters.

Reply:
OK! Thanks!